schema/me/notifications.js

```diff
@@ -46,6 +46,10 @@ const NotificationsFeedItemType = new GraphQLObjectType({
         },
       }),
     },
+    image: {
+      type: GraphQLString,
+      resolve: ({ object }) => object.artist.image_url,
```

Comment:
I don't think this is exactly what you need.

- We should be using the new `artists` part of artwork JSON, `artist` is deprecated.
- The `image_url` isn't actually a 'valid' URL, its the placeholder `:version` one.

We already give you a bunch of helpers using [Image](https://github.com/artsy/metaphysics/blob/3bfc10db1aa2c8813d91bff8e78d398a7cfb9a65/schema/image/index.js), so I think something like `artists.count > 0 ? Image.resolve(object.artists[0]) : null` maybe?

Then, you get all the usual methods you'd need for images, like crop and resize.